### PR TITLE
refactor: change deployment for kyverno on testing cluster

### PR DESCRIPTION
**Pull request by ricogu**

### CODE REVIEW

File: environments/core/applicationsets/argo-applicationset.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
       - cluster: core
         cluster-name: in-cluster
         overlay: overlays/core
-        targetRevision: ArgoCD-v2.4.16_AVP-v1.13.0
+        targetRevision: Config-change-kyverno
```
Comment (carslen) [inline environments/core/applicationsets/argo-applicationset.yaml:12]: Why changing the `targetRevision` for the ArgoCD setup itself?

Comment (ricogu) [reply environments/core/applicationsets/argo-applicationset.yaml:12]: in pr: https://github.com/catenax-ng/k8s-cluster-stack/pull/271

I need to apply some global configuration for deploying kyveron

**Review by carslen — CHANGES_REQUESTED** [review]
OK, got it. But the used Tag is totally wrong, from my pov. We've agreed on a Tag representing the ArgoCD setup and used versions. Maybe we should introduce something like a config iteration to cover changes like yours:

- `ArgoCD-v2.4.15-c1_AVP-v1.13.0` (_c1_ for config change 1)
- `ArgoCD-v2.4.15-i1_AVP-v1.13.0` (_i1_ for config iteration 1)
- `ArgoCD-v2.4.15-st1_AVP-v1.13.0` (_st1_ for system team (change) 1)
- ... (add your proposal here)

But Config-change-kyverno is a no-go for me. 

As we're pushing the Dev teams to use proper commit messages, I would like to invite you too to lead with good examples in our repositories as well. :)

**Comment by ricogu** [comment]
@carslen @hzierer I have to change the deployment strategu therefore I will postpone the sync with tag and use this PR for the changes

**Review by Siegfriedk — APPROVED** [review]

**Pull request merged**